### PR TITLE
 DQL.sql

### PR DESCRIPTION
**Pull request by SharifovDeveloper**

### CODE REVIEW

File: sql/DQL.sql [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+    AVG(B.Total_Price) >= ALL (
+        SELECT MAX(AvgTotalPrice)
+        FROM (
+            SELECT AVG(B2.Total_Price) AS AvgTotalPrice
+            FROM Booking B2
+            JOIN Room R2 ON B2.Room_Id = R2.Id
+            WHERE R2.Category_Id = R.Category_Id
+            GROUP BY R2.Category_Id
+        ) AS Subquery
+    );
```
Comment (diyorbekaxmatov) [inline sql/DQL.sql:172]: All very good !

**Review by diyorbekaxmatov — APPROVED** [review]

File: sql/DQL.sql (hunk 1 above)
Comment (SharifovDeveloper) [reply sql/DQL.sql:172]: Aha

File: sql/DQL.sql [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+            Employee_Id
+    ) AS MaxBooking
+ON
+    E.Id = MaxBooking.Employee_Id
+JOIN
+    Booking B
+ON
+    MaxBooking.Employee_Id = B.Employee_Id
+    AND MaxBooking.MaxTotalPrice = B.Total_Price;
+
```
Comment (Nabiddinov) [inline sql/DQL.sql:140]: This query is very different

**Review by Nabiddinov — APPROVED** [review]

File: sql/DQL.sql (hunk 2 above)
Comment (SharifovDeveloper) [reply sql/DQL.sql:140]: Different or Difficult ?

**Comment by Mirazyzz** [comment]
Looks good.

**Pull request merged**